player/character_body_souls_base.gd

```diff
@@ -327,6 +327,8 @@ func free_movement():
 	input_dir = Input.get_vector("move_left", "move_right", "move_up", "move_down")
 	var new_direction = calc_direction()
 	if new_direction:
+		if Input.is_action_pressed("dodge_dash"):
```

Comment:
While this works, it does feel a bit hacky. Any suggestion to improve the code while maintaining the behaviour is appreciated.

Reply:
So I'm still very hesitant that you aren't way behind from the current version, so ce I haven't been able to recreate your issue, and I'm not going to pull in the fix.

Please download the latest and see if the issue still persistents. I really don't to want to latch a problem that has already been patched 

Also, don't trust the version number v1.0.1 to mean much . I commit about twice a day small fixes and additions. If I changes that number everytime we'd be a about v1.0.52

Reply:
on the Pull request you can literally see i'm one commit ahead of master, i am completely up to date, the only thing i can think of is if you haven't pushed the latest version